rand_core/src/lib.rs

```diff
@@ -385,6 +385,7 @@ pub trait SeedableRng: Sized {
     /// [`getrandom`]: https://docs.rs/getrandom
     #[cfg(feature = "getrandom")]
     #[cfg_attr(doc_cfg, doc(cfg(feature = "getrandom")))]
+    #[track_caller]
     fn from_entropy() -> Self {
```

Comment:
The potential for failure is due to `getrandom`, so probably this shouldn't track the caller.